audio-worklet/src/noisegate-demo.js

```diff
@@ -187,6 +206,19 @@ class NoisegateDemo {
     }
   }
 
+  workletSelected() {
+    // Switching occurs 10ms into the future for thread synchronization.
```

Comment:
We scheduled switching between script processor and audio worklet implementations into the future in the bitcrusher for reasons having to do with that the script processor operates in a separate thread and at a slight delay, but I suppose I don't fully understand the mechanics of why this is necessary. I defer to @rtoy that the comment and scheduling is correct.